database/lock.go

```diff
@@ -68,7 +68,6 @@ func Lock(name string, duration time.Duration, owner string) (bool, time.Time) {
 func Unlock(name, owner string) {
 	pruneLocks()
 
-	t := cayley.NewTransaction()
 	it, _ := cayley.StartPath(store, name).Has("locked", "locked").Has("locked_by", owner).Save("locked_until", "locked_until").BuildIterator().Optimize()
```

Comment:
If the assumption is that this will only return one thing from the iterator, please verify it and log an error if your assumption is violated. The change to the transaction scope should be fine.